newIDE/app/src/KeyboardShortcuts/KeyboardShortcuts.spec.js

```diff
@@ -0,0 +1,100 @@
+import {
```

Comment:
Try to add `// @flow` and see if there are any errors.
Even for test, we try to have flow coverage so that we can get static typing and the safety that comes with it when doing refactorings in the future :)

Reply:
I would've done that, but I'll have to force-typecast the output of `makeKeyEvent` (`object` -> `any` -> `KeyboardEvent`) since two of the three tested functions take an event object as argument. The rest of it is fine. Should I do it?

Reply:
Let's still do it and add a `// $FlowIgnore - we intentionally create a fake KeyboardEvent` like comment to it :) 
Better to have a file typed even if there is a comment to ignore an error because it's a test - so not everything is perfectly simulated.